lib/pdfkit/html_preprocessor.rb

```diff
@@ -0,0 +1,23 @@
+class PDFKit
+  class HtmlPreprocessor
```

Comment:
Also the (unfortunate) existing style of this library is to use all caps in names, e.g., `PDFKit` so we should use `HTMLPreprocessor`.